### PR TITLE
release 8.0.7

### PR DESCRIPTION
**Pull request by thesuperzapper**
A patch release with lots of fixes:
* fix dockerfile code blocks in README
* only include checksum/config-pod-template for kubernetes_like executors
* give more information in value validation errors 
* prevent embedded postgres/redis being enabled at same time as external 
* use _helper variable in pod_template envFrom 
* include airflow.podAnnotations in jobs
   * partially closes https://github.com/airflow-helm/charts/pull/140
* add int64 to validation, so int variables set in bash work 
   * closes https://github.com/airflow-helm/charts/issues/136
* add missing pod labels to upgrade-db job 
* fix validation for wildcard ingress paths 
   * closes https://github.com/airflow-helm/charts/issues/144
* fix typo in connections example 
   * closes https://github.com/airflow-helm/charts/pull/148
* fix incorrect variable usage for variablesUpdate 
   * closes https://github.com/airflow-helm/charts/issues/139
*  add validation for airflow version with `airflow.legacyCommands` state
* add docs for using non-default airflow versions
* make `ingress.web/flower.tls.secretName` optional 
   * closes https://github.com/airflow-helm/charts/issues/41
* fix support for passwords with bash special characters 
   * closes https://github.com/airflow-helm/charts/pull/147

## Checklist

- [X] Commits are [signed](https://github.com/airflow-helm/charts/tree/main/CONTRIBUTING.md#sign-your-work)
- [X] Commits are [squashed](https://github.com/airflow-helm/charts/tree/main/CONTRIBUTING.md#squash-commits) (if appropriate)
- [X] Chart [version bumped](https://github.com/airflow-helm/charts/tree/main/CONTRIBUTING.md#versioning)
- [X] Chart [documentation updated](https://github.com/airflow-helm/charts/tree/main/CONTRIBUTING.md#documentation)
- [X] Passes [linting](https://github.com/airflow-helm/charts/tree/main/CONTRIBUTING.md#linting)

### CODE REVIEW

File: charts/airflow/templates/config/secret-config.yaml [hunk 1]
```diff
@@ -72,7 +72,7 @@ stringData:
   ## a bash command which echos the URL encoded value of $REDIS_PASSWORD
   ## NOTE: if $REDIS_PASSWORD is non-empty, prints `:${REDIS_PASSWORD}@`, else ``
   REDIS_PASSWORD_CMD: |-
-    echo ${REDIS_PASSWORD} | python3 -c "import urllib.parse; encoded_pass = urllib.parse.quote(input()); print(f\":{encoded_pass}@\") if len(encoded_pass) > 0 else None"
+    echo "${REDIS_PASSWORD}" | python3 -c "import urllib.parse; encoded_pass = urllib.parse.quote(input()); print(f\":{encoded_pass}@\") if len(encoded_pass) > 0 else None"
```
Comment (gsemet) [inline charts/airflow/templates/config/secret-config.yaml:75]: Actually it would be even safer to use directly is.environ instead of echo and input

Comment (thesuperzapper) [reply charts/airflow/templates/config/secret-config.yaml:75]: Its probably fine for now, but can you raise an issue to just use straight python, and put it in enhancements todo (see the kanban I made https://github.com/airflow-helm/charts/projects/2)

Comment (thesuperzapper) [reply charts/airflow/templates/config/secret-config.yaml:75]: Wait, it might not work with " now, I might have to fix it, lol

**Comment by thesuperzapper** [comment]
@gsemet any extra comments, so I can merge it and go to sleep?

**Comment by gsemet** [comment]
Sure, let’s merge and go to sleep !!

File: charts/airflow/templates/config/secret-config.yaml (hunk 1 above)
Comment (thesuperzapper) [reply charts/airflow/templates/config/secret-config.yaml:75]: @gsemet I raised an issue to track this enhancement idea https://github.com/airflow-helm/charts/issues/151

**Pull request merged**